contracts/actions/common/TakeFlashloan.sol

```diff
@@ -7,20 +7,23 @@ import { IERC3156FlashBorrower } from "../../interfaces/flashloan/IERC3156FlashB
 import { IERC3156FlashLender } from "../../interfaces/flashloan/IERC3156FlashLender.sol";
 import { FlashloanData } from "../../core/types/Common.sol";
 import { OPERATION_EXECUTOR, DAI, TAKE_FLASH_LOAN_ACTION } from "../../core/constants/Common.sol";
-import { FLASH_MINT_MODULE } from "../../core/constants/Maker.sol";
+import { MCD_FLASH } from "../../core/constants/Maker.sol";
 import { ProxyPermission } from "../../libs/DS/ProxyPermission.sol";
+import { ChainLogView } from "../../core/views/ChainLogView.sol";
 
 /**
  * @title TakeFlashloan Action contract
  * @notice Executes a sequence of Actions after flashloaning funds
  */
 contract TakeFlashloan is Executable, ProxyPermission {
   ServiceRegistry internal immutable registry;
+  ChainLogView internal immutable chainlogView;
   address internal immutable dai;
 
-  constructor(ServiceRegistry _registry, address _dai) {
+  constructor(ServiceRegistry _registry, address _dai, ChainLogView _chainlogView) {
```

Comment:
If `registry` is not used anywhere then just remove it. No need to provide additional argument during deployment and spend gas on the storage variable for it .